### ISSUE TITLE
The sample app makes it seems like the notifications request contains the hubLoginToken as a URL param

### ISSUE THREAD

**Issue by ssaloisbellerose** [BroadSoft-Xtended/DeveloperPortalDocs#20]
In the HelloWorld sample app, there is this line:

// const hubLogintoken = req.query.hubLogintoken;

here:

// This is where you set the notifications number to be shown
router.post('/:appName/notifications', function(req, res) {
  // const hubLogintoken = req.query.hubLogintoken;
  // This is used to identify your user. You should save the token and associate it to a user when you authenticate the user
  console.log('We are requesting the notifications count');
  res.send(200, {count: 99});
});

Because of this, I tried to get the hubLoginToken as a query parameter to the notifications route. This line should be removed to avoid confusion and should ideally show how we can get the "auth" value from the notifications route.

**Comment by joncodo** [comment]
You are totally right. We will update the sample apps with the new release of the hub api that is coming soon
